pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/legacy/fraser/TBU2361_Generated.kt

```diff
@@ -0,0 +1,50 @@
+package no.nav.pensjon.brev.maler.legacy.fraser
+
+import no.nav.pensjon.brev.api.model.maler.legacy.PE
+import no.nav.pensjon.brev.maler.legacy.vedtaksdata_beregningsdata_beregningufore_belopokt
+import no.nav.pensjon.brev.maler.legacy.vedtaksdata_beregningsdata_beregningufore_belopredusert
+import no.nav.pensjon.brev.maler.legacy.vedtaksdata_beregningsdata_beregningufore_beregningytelseskomp_uforetrygdordiner_avkortningsinformasjon_forventetinntekt
+import no.nav.pensjon.brev.maler.legacy.vedtaksdata_beregningsdata_beregningufore_beregningytelseskomp_uforetrygdordiner_avkortningsinformasjon_utbetalingsgrad
+import no.nav.pensjon.brev.maler.legacy.vedtaksdata_beregningsdata_beregningufore_uforetrygdberegning_uforegrad
+import no.nav.pensjon.brev.maler.legacy.vedtaksdata_kravhode_kravarsaktype
+import no.nav.pensjon.brev.model.*
+import no.nav.pensjon.brev.template.*
+import no.nav.pensjon.brev.template.Language.*
+import no.nav.pensjon.brev.template.dsl.*
+import no.nav.pensjon.brev.template.dsl.expression.*
+
+
+data class TBU2361_Generated(
+    val pe: Expression<PE>,
+) : OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+		//[TBU2361NN, TBU2361, TBU2361EN]
+		paragraph {
+			textExpr (
+				Bokmal to "Du har tidligere meldt fra om en inntekt på ".expr() + pe.vedtaksdata_beregningsdata_beregningufore_beregningytelseskomp_uforetrygdordiner_avkortningsinformasjon_forventetinntekt().format() + " kroner i år.",
+				Nynorsk to "Du har tidlegare meldt frå om ei inntekt på ".expr() + pe.vedtaksdata_beregningsdata_beregningufore_beregningytelseskomp_uforetrygdordiner_avkortningsinformasjon_forventetinntekt().format() + " kroner for dette kalenderåret.",
+				English to "You have previously reported an income of NOK ".expr() + pe.vedtaksdata_beregningsdata_beregningufore_beregningytelseskomp_uforetrygdordiner_avkortningsinformasjon_forventetinntekt().format() + " for this calendar year.",
+			)
+
+			// Henger sammen med 2362 på engelsk Ikke gjør noe med dette før all konvertering er ferdig siden vi vet ikke hvordan frasen brukes enda.
```

Comment:
Betyr dette at det er riktig (for nå) at det ikke skal stå noe på bokmål og nynorsk?

Reply:
yep. Kunne evt lagd en tredje frase som samler de, men jeg synest det blir rotete om vi starter å fikse på fraser før alt er flyttet over.